supervisor/resolution/evaluations/trust.py

```diff
@@ -0,0 +1,35 @@
+"""Evaluation class for Job Conditions."""
```

Comment:
```suggestion
"""Evaluation class for content-trust."""
```
The filename and the resulting slug should be `content_trust` as well